contracts/axelar-gateway/src/testutils.rs

```diff
@@ -20,27 +20,27 @@ const DESTINATION_ADDRESS: &str = "0x4EFE356BEDeCC817cb89B4E9b796dB8bC188DC59";
 #[derive(Clone, Debug)]
 pub struct TestSignerSet {
     pub signers: std::vec::Vec<SigningKey>,
-    pub signer_set: WeightedSigners,
+    pub signers_set: WeightedSigners,
```

Comment:
Should be `signer_set` since it's a single set of signers, and not a `Vec<WeightedSigners>`

Alternatively, call this `signers` and change existing `signers` to `signer_keys`

Reply:
updated `signers_set: WeightedSigners` to `signers`  and `signers: std::vec::Vec<SigningKey>` to `signer_keys`